packages/relay-compiler/codegen/RelayFileWriter.js

```diff
@@ -24,7 +24,7 @@ const graphql = require('graphql');
 const invariant = require('invariant');
 const nullthrows = require('nullthrows');
 const path = require('path');
-const writeRelayGeneratedFile = require('./writeRelayGeneratedFile');
+const {writeRelayGeneratedFile} = require('./writeRelayGeneratedFile');
```

Comment:
why this?

Reply:
So I could export the function we want to test 🤔